userguide/tutorials/development.adoc

```diff
@@ -146,11 +148,11 @@ Once the repository is cloned, you need to create an Eclipse project. For this,
       </pluginExecution>
    </pluginExecutions>
 </lifecycleMappingMetadata>
+----
 
+. Go to `Window > Preferences > Maven > Lifecycle Mappings`. Click *Browse* and select the path of the `lifecycle-mapping-metadata.xml` file. Click `Apply and Close`.
```

Comment:
```suggestion
. Go to `Window > Preferences > Maven > Lifecycle Mappings`. Click *Browse* and select the path of the `lifecycle-mapping-metadata.xml` file. Click *Apply and Close*.
```

Reply:
I haven't yet addressed this in our "style guide," but the standard I have been using is to bold face what gets clicked or selected. 

I am pretty confident about that one. What I'm not so confident about is when to use the code-type font (like Courier) in our in-line text. So far I've been using it for URLs, file names, settings, etc.

Reply:
Fixed here and in a couple of other places.